package.json

```diff
@@ -73,12 +74,6 @@
         "lint-staged": ">=10.5.1",
         "prettier": "^2.1.2",
         "rimraf": "^3.0.0",
-        "rollup": "^2.22.1",
-        "rollup-plugin-babel": "^4.4.0",
-        "rollup-plugin-commonjs": "^10.1.0",
-        "rollup-plugin-dts": "^1.4.9",
-        "rollup-plugin-node-resolve": "^5.2.0",
-        "rollup-plugin-typescript2": "^0.29.0",
```

Comment:
Nice.

Reply:
Pure `tsc` is a better choice for purely nodejs apps. I originally wanted rollup so I could compile all the dependencies into one `.js` file that could be quickly installed and executed in the main `posthog` repo. There were some problems with that:
- Some dependencies didn't want to be compiled into one `.js` file due to a bunch of random reasons, so I had to keep them as separate `require` statements.
- Eventually it was easier to just keep all dependencies outside the `index.js` and require them in, only compiling all of `posthog-plugin-server` into one `.js`
- Introducing piscina workers, which require loading a separate `.js` file that gets started in a new thread, broke that system... and it was easier to just remove rollup.